packages/gatsby/lib/query-runner/compiler-helpers.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import fs from "fs"
+import path from "path"
+import { FileIRParser, FileWriter, IRTransforms } from "relay-compiler"
+import { buildASTSchema, parse } from "graphql"
+import { store } from "../redux"
```

Comment:
this is mostly borrwed from `relay-compiler-webpack-plugin` i'd be nice to do the query evaluation during the webpack compile cycle but I see why it's done this way...maybe a webpack plugin?